### ISSUE TITLE
jsf throws on object without properties, but using minProperties

### ISSUE THREAD

**Issue by JonathanMontane** [json-schema-faker/json-schema-faker#157]
The following schema is valid:

``` js
{
  "minProperties": 1
}
```

but throws in jsf due to jsf requiring the schema having either:
- `properties`
- `patternProperties`
- `additionalProperties`

To my knowledge, no such constraint exists in the draft of JSON Schema.

A possible generated value is:

``` js
{
  "Utf6f0000": {}
}
```

**Comment by ducin** [comment]
How about this one - could it be closed after #159 is merged into master, just as in #158? @pateketrueke, @JonathanMontane?

**Comment by pateketrueke** [comment]
Proudly fixed! :tada:
